src/components/backupWallet/ExportWalletData.js

```diff
@@ -0,0 +1,151 @@
+
+  return (
+    <View style={styles.borderedBox}>
+      <View style={styles.boxAvatarContainer}>
+        <Image source={avatarSource} style={styles.avatar} />
+      </View>
+      <Section.Text fontSize={18} fontFamily="Roboto Slab" fontWeight="bold" style={styles.boxTitle}>
+        {title}
+      </Section.Text>
+      <Section.Text fontSize={13} letterSpacing={0.07} color={theme.colors.lighterGray}>
+        {content}
+      </Section.Text>
+      <TouchableOpacity onPress={copyToClipboard} activeOpacity={1} style={styles.boxCopyIconWrapper}>
+        <View style={styles.copyIconContainer}>
+          <Icon name="copy" size={32} color={theme.colors.surface} />
+        </View>
+        <Section.Text fontSize={10} fontWeight="medium" color={theme.colors.primary}>
+          {copyButtonText}
+        </Section.Text>
+      </TouchableOpacity>
+    </View>
+  )
+}
+
+const ExportWalletData = ({ navigation, styles, theme }: BackupWalletProps) => {
+  const { navigate } = navigation
+  const gdstore = GDStore.useStore()
+  const { avatar } = gdstore.get('profile')
+  const avatarSource = useMemo(() => (avatar ? { uri: avatar } : unknownProfile), [avatar])
+
+  const handleGoHome = useOnPress(() => navigate('Home'), [navigate])
+
+  return (
+    <Wrapper style={styles.wrapper}>
+      <NavBar title="EXPORT MY WALLET" goBack={handleGoHome} />
+      <Section grow justifyContent="space-around">
+        <BorderedBox
+          styles={styles}
+          theme={theme}
+          title="My Wallet Private Key"
+          content={privateKey}
+          avatarSource={avatarSource}
+          copyButtonText="Copy Key"
+        />
+        <BorderedBox
+          styles={styles}
+          theme={theme}
+          title="Fuse Network RPC Address"
+          content={web3ProviderUrl}
+          avatarSource={avatarSource}
+          copyButtonText="Copy Address"
+        />
+      </Section>
+      <CustomButton onPress={handleGoHome}>Done</CustomButton>
+    </Wrapper>
+  )
+}
+
+const styles = ({ theme }) => ({
+  wrapper: {
+    backgroundImage: 'none',
+    backgroundColor: 'none',
+    padding: 0,
+  },
+  borderedBox: {
+    borderWidth: 1,
+    borerStyle: 'solid',
+    borderColor: theme.colors.lighterGray,
+    borderRadius: 5,
+    height: getDesignRelativeHeight(123, false),
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    position: 'relative',
+  },
+  boxTitle: {
+    marginBottom: getDesignRelativeHeight(10, false),
+  },
+  boxAvatarContainer: {
+    height: getDesignRelativeHeight(88, false),
+    width: getDesignRelativeHeight(88, false),
+    position: 'absolute',
+    top: -getDesignRelativeHeight(44, false), // half of height
+    padding: getDesignRelativeHeight(10, false),
+    backgroundColor: theme.colors.surface,
+  },
+  avatar: {
+    height: getDesignRelativeHeight(68, false),
+    width: getDesignRelativeHeight(68, false),
```

Comment:
designrelativewidth

Reply:
avatar is a circle element and the getDesignRelativeHeight and getDesignRelativeWidth changing the value differently and for mobile devices the element becomes oval shape.
So I am using the getDesignRelativeHeight for both to keep the element with circle shape for all devices.